src/auth/token.strategy.ts

```diff
@@ -9,7 +9,7 @@ export class TokenStrategy extends PassportStrategy(
   'discordGithub-strategy',
 ) {
   constructor(private config: ConfigService) {
-    super({});
+    super({ tokenHeader: 'Client-Token' });
```

Comment:
Could you explain a bit more about this and where it comes from?

Reply:
I was thinking the client would send in 

```
@ApiHeader({ name: 'User-Uid', required: true })
@ApiHeader({ name: 'Platform', required: true })
```

Reply:
Thats the configuration of the [Passport-Strategy](http://www.passportjs.org/packages/passport-unique-token/) for validating the actual api-key. The header values for identifying the author of the object are validated after the token validation was successfull

Reply:
ok makes sense, thank you 👍